src/calendar/index.tsx

```diff
@@ -194,7 +192,7 @@ const Calendar = (props: CalendarProps) => {
           marking={weekNumberMarking.current}
           // state='disabled'
           theme={theme}
-          testID={`${WEEK_NUMBER}-${weekNumber}`}
+          testID={`${testID}.week_${weekNumber}`}
```

Comment:
Why is it called "week" and not "week_number"?

Reply:
Because it's a week_[number], it's like I do with day_[date]
Let me know if you still prefer weekNumber

Reply:
I prefer "week_number" or "weekNumber" since we have a `Week` component and because it's actually on a day component, so it can be confusing.

Reply:
Fair enough, fixed